src/chart/generateCategoricalChart.tsx

```diff
@@ -2078,19 +2097,48 @@ export const generateCategoricalChart = ({
       return result;
     };
 
+    renderActiveBar = ({ item, childIndex }: any) => {
+      const result = [];
+      const { key } = item.props;
+      const { activeBar, dataKey } = item.item.props;
+
+      /**
+       * Default to inheriting props from the parent bar
+       */
+      const inheritedProps = {
+        ...item.item.props,
+        ...item.props.data[childIndex],
+      };
+
+      const barProps = {
+        index: childIndex,
+        dataKey,
+        fill: getMainColorOfGraphicItem(item.item),
+        strokeWidth: 2,
+        stroke: '#fff',
```

Comment:
Does this mean the outline of the bar will always be black and a fixed width?

people are probably going to want to customize these - do we have ability to modify them at all?

Reply:
Yes we have the ability to modify. Below these lines I spread in props from the parent bar, then props from the activeBar, in that order.

So declaring attributes on `activeBar` will take precedent
<img width="272" alt="Screenshot 2023-09-13 at 9 08 19 PM" src="https://github.com/recharts/recharts/assets/24420033/8d217e93-23c0-4d64-88ca-f47b5394300c">

No attributes on `activeBar` will then inherit from values declared on the parent bar
<img width="311" alt="Screenshot 2023-09-13 at 9 07 46 PM" src="https://github.com/recharts/recharts/assets/24420033/bd3acea0-096a-4be5-b17a-ceba39c75aae">

And finally if both `activeBar` and parent bar don't have these values defined then we'll hit these two hardcoded values you are referencing

Reply:
cool thank you